testcontainers/src/runners/async_runner.rs

```diff
@@ -63,8 +63,17 @@ where
             .map(|(key, value)| format!("{key}:{value}"))
             .collect();
 
+        let labels = HashMap::<String, String>::from_iter(
+            container_req
+                .labels()
+                .iter()
+                .map(|(key, value)| (key.into(), value.into()))
+                .chain([("managed-by".to_string(), "testcontainers".to_string())]),
```

Comment:
Actually, to avoid any collisions  and possible conflicts with user's code. I'd suggest to use predefined prefix for **all** our labels, including `managed-by`

- `org.testcontainers.managed-by` 

So we can claim that `org.testcontainers.*` prefix is dedicated for usage by testcontainers and it will be unlikely to collide

WDYT?

Reply:
I don't have a strong enough argument *against* it, honestly, other than "the semantics / intent / meaning of `managed-by` or `app.docker.io/managed-by` *feels* clearer than `org.testcontainers.managed-by`", but "✨vibes✨" is absolutely not a valid reason to argue in this context.

Plus, as I was thinking about it, the counter-point advocate part of my brain floated it as a really nice / clean way to configure reaper behavior in the future, e.g.:

```rust
let managed_by = if !reusable && !ryuk {
    // reaper behavior controlled by standard `Drop` impl
    "testcontainers"
} else if reusable && !ryuk {
    // reaper behavior explicitly controlled by user code
    let managed_by = env!("CARGO_PKG_NAME");
} else if !reusable && ryuk {
    // reaper behavior controlled by `ryuk`
    let managed_by = "ryuk";
} else {
    // etc...
};

let labels = HashMap::<String, String>::from_iter(
    container_req
        .labels()
        .iter()
        .map(|(key, value)| (key.into(), value.into()))
        .chain([("org.testcontainers.managed-by".into(), managed_by.into())]),
);
```

So... yeah. Sold. Implemented and pushed 😁